src/table/utils/selections-utils.ts

```diff
@@ -1,11 +1,38 @@
-export const SelectionStates = {
-  SELECTED: 'selected',
-  POSSIBLE: 'possible',
-  EXCLUDED: 'excluded',
-  INACTIVE: 'inactive',
+import { stardust } from '@nebula.js/stardust';
+import {
+  TableCell,
+  SelectionState,
+  Action,
+  ExtendedSelectionAPI,
+  ActionPayload,
+  AnnounceFn,
+  SelectedRows,
+  ContextValue,
+} from '../../types';
+
+// eslint-disable-next-line no-shadow
+export enum SelectionStates {
```

Comment:
I have no idea why lint thinks this is shadowing. The error says that it has already been defined on line 14.... redefined on the line it is being defined? 👍

Reply:
let me check

Reply:
Found it: https://stackoverflow.com/questions/63961803/eslint-says-all-enums-in-typescript-app-are-already-declared-in-the-upper-scope

The normal eslint rule had to be disabled and enabled as a typescript specific rule. Should be fine to do already since we shouldn't be adding any js code anymore